### PR TITLE
Add 'test' snippet

### PR DESCRIPTION
**Pull request by raphaeljlps**

### CODE REVIEW

File: snippets/language-go.cson [hunk 1]
```diff
@@ -107,3 +107,6 @@
   'goroutine function':
     'prefix': 'gf'
     'body': 'go ${1:func}($0)'
+  'test':
```
Comment (svanharmelen) [inline snippets/language-go.cson]: Would be nice if we can change this to 'test function' as this is the description you'll see in the autocomplete popup.

File: snippets/language-go.cson [hunk 2]
```diff
@@ -107,3 +107,6 @@
...
+    'prefix': 'test'
```
Comment (svanharmelen) [inline snippets/language-go.cson]: And to keep things a little inline with each other, could we change this to `tf`?

**Comment by svanharmelen** [comment]
@raphaeljlps 2 small suggestions, otherwise nice snippet... Thx!

**Comment by svanharmelen** [comment]
Wow, your fast :wink: LGTM :+1: 

Thanks!

**Comment by raphaeljlps** [comment]
Good suggestions @svanharmelen 
I have already made the changes

**Comment by 50Wliu** [comment]
Thanks!

**Comment by raphaeljlps** [comment]
Thank you @50Wliu

**Pull request merged**